l1-contracts/src/governance/GSE.sol

```diff
@@ -274,8 +279,8 @@ contract GSECore is IGSECore, Ownable {
    * Now she cannot deposit through D AT ALL, since she is already in D's effective attesters.
    * But she CAN go back and deposit directly into C, with _moveWithLatestRollup = false.
    *
-   * @param _attester     - The attester address of the attester
-   * @param _withdrawer   - The withdrawer address of the attester
+   * @param _attester     - The attester address on behalf of which the deposit is made.
+   * @param _withdrawer   - Address to which the staking asset will be sent when withdrawing from `_attester`.
```

Comment:
This change is not correct. It is not the location of the funds, it is the controller of them.

Reply:
Ok ok, I didn't look much beyond GSE so did not understand the nuances much. Just felt like the comment is not that helpful. Could you push a commit with how you think it should be documented?

Reply:
Sure will do. But will also create a pr since I realised there are some more foundry formatting settings that apply the line-width to comments itself as well.